### PR TITLE
Added lightning logger to API reference

### PR DESCRIPTION
**Pull request by annaelisalappe**
# Summary

This PR adds the lightning logger to the API reference in the docs. 

**Related issue :**

### CODE REVIEW

**Review by matbun — CHANGES_REQUESTED** [review]
I would move it under torch module to make it compliant with the itwinai module structure

File: docs/python-api/itwinai.loggers.rst [hunk 1]
```diff
@@ -1,9 +1,8 @@
 itwinai.loggers
-===============
+-----------------
```
Comment (matbun) [inline docs/python-api/itwinai.loggers.rst:2]: I have not checked, but are you sure that this is not going to mess with the headings hierarchy ?

File: docs/python-api/itwinai.torch.modules.rst [hunk 2]
```diff
@@ -64,3 +64,12 @@ trainer.py
    :undoc-members:
    :show-inheritance:
    :member-order: bysource
+
+
+loggers.py
```
Comment (matbun) [inline docs/python-api/itwinai.torch.modules.rst:69]: please sort the module names

**Review by matbun — CHANGES_REQUESTED** [review]

File: docs/python-api/itwinai.loggers.rst (hunk 1 above)
Comment (annaelisalappe) [reply docs/python-api/itwinai.loggers.rst]: Changed it :)

**Pull request merged**